browser/brave_stats/brave_stats_updater.cc

```diff
@@ -463,7 +463,8 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   registry->RegisterIntegerPref(kLastCheckMonth, 0);
   registry->RegisterStringPref(kLastCheckYMD, std::string());
   registry->RegisterStringPref(kWeekOfInstallation, std::string());
-  registry->RegisterTimePref(kBraveWalletPingReportedUnlockTime, base::Time());
+  registry->RegisterTimePref(kBraveWalletPingReportedUnlockTime,
+                             base::Time::Now());
```

Comment:
So this is a similar work around for `kBraveWalletP3ALastReportTime` in #12480?

Will this only help profiles upgrading from 1.35? It seems like this would have no effect if they're already launched a 1.36 release?

Reply:
totally separate issue, and yes you are correct! This will only fix upgrades to 1.36.x. With the current code, if they already used their wallet in 1.35.x, upgraded to 1.36.x and launched their browser, the damage has already been done. This will only fix future upgrades to 1.36.x (i.e. Linux, which is usually delayed).